scripts/install_infra.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#
+# Builds docker image and install the ipc-scripts required to conveniently
+# deploy the infrastructure for IPC subnets.
+rm -rf ./lotus
+git clone https://github.com/consensus-shipyard/lotus.git
```

Comment:
Do we want to check out master or track a particular release? Master is probably fine, but just checking.

Reply:
I am a bit wary about fixing the release here for now just in case we forget to change it after an upgrade, but maybe we should? Both approaches have their own drawbacks

Reply:
Yeah... I think it's fine to leave it as is; we'll just need to be careful to update the guide if the CLI changes. And to not break the build, I suppose.